src/components/BookCard.jsx

```diff
@@ -5,7 +5,12 @@ import Style from './BookCard.module.css';
 import 'react-circular-progressbar/dist/styles.css';
 
 /* eslint-disable jsx-a11y/anchor-is-valid, react/prop-types,react/jsx-curly-brace-presence */
-function BookCard({ title, category, author }) {
+const Button = ({ removeBookHandler }) => (
+  <button type="button" onClick={removeBookHandler} className={`${Style.RemoveBtn}`}> Remove Book</button>
+);
+function BookCard({
+  title, category, author, removeBookHandler,
+}) {
```

Comment:
- [ ] You have done great so far, but there are still some requirements you are yet to implement, 
- [ ] As pointed out by the previous about your remove button, I can see you added a `Remove book` button at the end of every book card. looking at the UI you can see there is already a `remove` in the design, that's where you are meant to add the functionality to remove a book, you are not meant to create a new button for that. :+1: :+1: 

- - Screenshot:
![Selection_001](https://github.com/habitoz/awesome-book-react/assets/107632780/9e0132e3-1b3c-4872-98a0-52e2232bc9a5)
- - Code Screenshot:
![Selection_002](https://github.com/habitoz/awesome-book-react/assets/107632780/a2d87141-6338-4bba-b6cc-3c4efe96bd50)

- [ ] As I pointed out above, that's where you are meant to add the functionality to remove a book. :+1: :+1: Hope that's clear enough :+1: